style.css

```diff
@@ -0,0 +1,193 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: 'Open Sans', sans-serif;
+  font-family: 'Poppins', sans-serif;
+  font-family: 'Roboto', sans-serif;
+  font-family: 'Space Mono', monospace;
+}
+
+/* MOBILE VERSION */
+@media only screen and (min-width: 375px) {
+  #nav-bar {
+    display: flex;
+    width: 80%;
+    justify-content: space-between;
+    margin: auto;
+    align-items: center;
+    background-image: url(./Images/header-background-mobile.png);
+    position: sticky;
+    top: 0;
+    z-index: 4;
+    padding-top: 20px;
+  }
+
+  #side-navbar-container {
+    display: none;
+  }
+
+  #union-img:hover {
+    opacity: 0.8;
+  }
+
+  #union-img:active {
+    opacity: 3;
+  }
+
+  #logo-image {
+    width: 5rem;
+    height: 1.25rem;
+  }
+
+  #vector-image {
+    border: 2px solid rgb(92, 54, 166);
+    width: auto;
+    height: 30px;
+  }
+
+  main {
+    height: 100vh;
+    background-image: url(./Images/header-shapes\ mobile.png);
+    background-size: cover;
+  }
+
+  #main-container {
+    display: flex;
+    flex-direction: column;
+    width: 70%;
+    margin: auto;
+    padding-top: 100px;
+  }
```

Comment:
- Hey, great job with your styles, however, there are some adjustments to add to make your design better. see the suggestions below:

   - Let's improve on the padding as I've edited your styles to be close to the design. kindly update your styles.
   - Remove the margin auto, as this is making your deign not consistent.
   - the width should be 100%, I commented it out as by default its 100% inherited from its parent container.
   - You need to align the item using flex start so they be on the same line on the left.
   - 


```
#main-container {
    display: flex;
    flex-direction: column;
    /* width: 70%; */
    /* margin: auto; */
    /* padding-top: 100px; */
    padding: 100px 24px;
    align-items: flex-start;
}
```

*your design*
![Screenshot from 2023-04-27 08-16-37](https://user-images.githubusercontent.com/59397159/234791148-86bcb54e-d340-4f06-a0bc-d4334d6d9f44.png)

*After you improve the styling as I've suggested, your design would look better.*
**You'd also need to add some little margin top and bottom or margin-top, and margin-bottom to the 
`LET'S CONNECT, h1, paragraph`**

Reply:
Fixed.